zone/corpse.h

```diff
@@ -142,6 +147,9 @@ class Corpse : public Mob {
 	int32		player_kill_item; /* Determines if Player Kill Item */
 	uint32		corpse_db_id; /* Corpse Database ID (Player Corpse) */
 	uint32		char_id; /* Character ID */
+	uint32		consent_group_id = 0; /* consented group id */
```

Comment:
I think we need to re-evaluate if we should allow this feature (non-static data member initializers), I think there was some issues with earlier GCC versions?

Reply:
This is technically C++11 - which is what we're at; so I'm good with it if everyone else is, its reduces constructor redundancy